stable/prometheus/templates/node-exporter-daemonset.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "prometheus.nodeExporter.fullname" . }}
 spec:
+  updateStrategy:
```

Comment:
This should be first added in `values.yaml` and then used here.

refer other sections such as `pod.spec.resources`, how they have been defined in the .values.yaml file and how they have been used here
```
 resources:
{{ toYaml .Values.nodeExporter.resources | indent 12 }}
```

Please bump update the chart version

Reply:
Made the changes. Also tested with `helm template`:
```
  updateStrategy:
    rollingUpdate:
      maxUnavailable: 1
    type: RollingUpdate
```